var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -39,8 +39,9 @@ def install(self, spec, prefix):
             make_defs += ['BUILD_LAPACK_DEPRECATED=1']
 
         # Add support for OpenMP
-        # Note: Make sure your compiler supports OpenMP
-        if '+openmp' in spec:
+        # Note: Apple's Clang 7.3.0 does not support OpenMP.
+        # For now, disable OpenMP if we compile with clang.
+        if ('+openmp' in spec) and (self.compiler.name != 'clang'):
             make_defs += ['USE_OPENMP=1']
```

Comment:
`print` does not print to the terminal, there must be a command to display a warning, but i never used it.